### ISSUE TITLE
Not working for cshtml files in version 15.9.7

### ISSUE THREAD

**Issue by solomonshaffer** [madskristensen/BrowserReloadOnSave#17]
Does not seem to be working on VS 15.9.7

https://www.screencast.com/t/4Uk76j7w00

**Comment by solomonshaffer** [comment]
Looks like this working for js and css files. But not cshtml files.

**Comment by ghost** [comment]
Duplicate of https://github.com/madskristensen/BrowserReloadOnSave/issues/7

**Comment by mdora7** [comment]
Yes it doesn't working but what i do is go to Tools->Options->Keyboard then search Linked Browser(exact name is OtherContextMenus.BrowserLink.RefreshLinkedBrowsers) and assign CTRL+S key to it

**Comment by ghost** [comment]
@solomonshaffer and @mdora7 I sent a PR in September 2019 which was already merged. Problem should be gone. 

Note that you must install the new version from http://vsixgallery.com/extension/2d8aa02a-8810-421f-97b9-86efc573fea3/ 
Version on Visual Studio Marketplace is not updated AFAIK.
